src/util.ts

```diff
@@ -1,5 +1,17 @@
 import { AxiosError } from 'axios';
 
+export function safeJsonParse(input: any, defaultValue: unknown): unknown {
+  if (!input) {
```

Comment:
Do we need this handling? Or will it still work the same if we remove it?

Reply:
Almost the same. For example:
- `JSON.parse(undefined)` throws (the same resulting behavior)
- `JSON.parse(null)` returns `null` (different behavior -> won't return the default value but `null` instead)

Reply:
`null` is a valid JSON, so it's correct

Reply:
Ok, I will drop it if you believe it makes the behavior more expected/less confusing.

Reply:
Dropped.